Added script to play the mob animation

utsav132 opened the pull request:
Added script to play the mob animation, along with its explanation in the docs.

In 'Your first 2D game' in the Getting Started part, the docs tell you how to randomly select mob animation (in creating the enemy), but don't show you how to play it. The '$AnimatedSprite2D.play()' part was missing the the script. This, along with it's docs, has been added in this pull request.

skyace65 commented:
Since this PR changes demo project code please open a corresponding PR in the demo projects repository. https://github.com/godotengine/godot-demo-projects

utsav132 commented:
I have corrected the code in the godot-demo-projects repository. I added the line to play animation at the end, which is in sync with this PR. 

It's in the following PR:
Added line to play animation at the end. #1175
https://github.com/godotengine/godot-demo-projects/pull/1175

Calinou reviewed (approved):
Thanks!
